doc/source/force-constants.rst

```diff
@@ -9,6 +9,86 @@ eigenvectors at any arbitrary q via Fourier interpolation.
 
 .. contents:: :local:
 
+.. _fc_format:
+
+Force Constants Format
+----------------------
+
+**Description**
+
+The :py:attr:`ForceConstants.force_constants <euphonic.force_constants.ForceConstants.force_constants>` attribute contains the force constants matrix
+
+.. math::
+
+  \phi_{\alpha, {\alpha}'}^{\kappa, {\kappa}'} =
+  \frac{\delta^{2}E}{{\delta}u_{\kappa,\alpha}{\delta}u_{{\kappa}',{\alpha}'}}
+
+This describes the change in total energy when atom :math:`\kappa` is displaced in direction :math:`\alpha` and atom :math:`\kappa\prime` is displaced in direction :math:`\alpha\prime`.
+In Euphonic the force constants are stored in 'compact' form, which means that the minimum required information is stored.
+Using equivalent vectors, the force constants between every atom in the unit cell, and every atom in the supercell are all that is needed,
+so only :math:`N(3n)^2` values need to be stored rather than :math:`(3Nn)^2` as in the 'full' square force constants matrix.
```

Comment:
This (and below) would be easier to follow if it has been directly stated that _N_ is the number of unit cells in the supercell, and _n_ is the number of atoms in a unit cell.